rootfs-builder/rootfs.sh

```diff
@@ -190,9 +190,10 @@ if [ -n "${USE_DOCKER}" ] ; then
 		--env GOPATH="${GOPATH}" \
 		--env KERNEL_MODULES_DIR="${KERNEL_MODULES_DIR}" \
 		-v "${script_dir}":"/osbuilder" \
-		-v "${ROOTFS_DIR}":"/rootfs" \
+		-v "${ROOTFS_DIR}":"${ROOTFS_DIR}" \
 		-v "${kernel_mod_dir}":"${kernel_mod_dir}" \
 		-v "${GOPATH}":"${GOPATH}" \
+		--privileged \
```

Comment:
debootstrap needs chroot Is there another way of doing this?

Reply:
Is it mounting any special device or why chroot need it ?

Reply:
I think debootstrap first downloads the rootfs then chroots into it then mounts file systems and then installs packages.

Reply:
There are `fakeroot` and `fakechroot` packages. I had a quick play around but they didn't allow me to run `debootstrap` successfully ;(

Reply:
I will go through them and get back soon.

Thanks